### ISSUE TITLE
Invalid Provider Server Combination

### ISSUE THREAD

**Issue by mouellet** [jfrog/terraform-provider-artifactory#728]
**Describe the bug**

When trying to update from version 7.7.0 to 7.8.0, I'm getting this error:
```
╷
│ Error: an object with 4 attributes is required (3 given)
│ 
│   with provider["registry.terraform.io/jfrog/artifactory"],
│   on provider.tf line 1, in provider "artifactory":
│    1: provider "artifactory" {
│ 
╵


Invalid Provider Server Combination

  with provider["registry.terraform.io/jfrog/artifactory"],
  on provider.tf line 1, in provider "artifactory":
   1: provider "artifactory" {

The combined provider has differing provider schema implementations across
providers. Provider schemas must be identical across providers. This is
always an issue in the provider implementation and should be reported to the
provider developers.

Provider schema difference:   &tfprotov5.Schema{
  	Version: 0,
  	Block: &tfprotov5.SchemaBlock{
  		Version: 0,
  		Attributes: []*tfprotov5.SchemaAttribute{
  			&{
  				... // 5 identical fields
  				Computed:        false,
  				Sensitive:       true,
- 				DescriptionKind: s"PLAIN",
+ 				DescriptionKind: s"MARKDOWN",
  				Deprecated:      false,
  			},
- 			&{
- 				Name:        "api_key",
- 				Type:        s"tftypes.String",
- 				Description: "API token. Projects functionality will not work with any auth method other than access tokens",
- 				Optional:    true,
- 				Sensitive:   true,
- 				Deprecated:  true,
- 			},
  			&{
  				... // 5 identical fields
  				Computed:        false,
  				Sensitive:       false,
- 				DescriptionKind: s"PLAIN",
+ 				DescriptionKind: s"MARKDOWN",
  				Deprecated:      false,
  			},
  			&{
  				Name:            "url",
  				Type:            s"tftypes.String",
- 				Description:     "",
+ 				Description:     "Artifactory URL.",
  				Required:        false,
  				Optional:        true,
  				Computed:        false,
  				Sensitive:       false,
- 				DescriptionKind: s"PLAIN",
+ 				DescriptionKind: s"MARKDOWN",
  				Deprecated:      false,
  			},
  		},
  		BlockTypes:  nil,
  		Description: "",
  		... // 2 identical fields
  	},
  }
```

Here's my configuration:
```
provider "artifactory" {
  url          = "https://<redacted>.jfrog.io/artifactory"
  access_token = var.artifactory_access_token
}
```
I'm still getting the error when trying different configuration options, like:
- setting `check_license = false` explicitly ,
- Using`ARTIFACTORY_URL` and `ARTIFACTORY_ACCESS_TOKEN` environment variable instead.

**Additional context**
- artifactory version: 7.58.1
- terraform version: 1.3.9
- terraform provider version: 7.8.0

**Comment by danielmkn** [comment]
@mouellet, do you have `api_key` set in your vars?

**Comment by mouellet** [comment]
Hi @danielmkn, no I'm using the Access Token authentication method...

**Comment by danielmkn** [comment]
I see, I can reproduce it now.

**Comment by danielmkn** [comment]
Fix released in 7.10.0. Thanks for the ticket!
